2009-08-16  Yavor Doganov  <yavor@gnu.org>

	* Documentation/coding-standards.texi:
	* Documentation/manual/manual.texi:  Use unambiguous
	  @setfilename; and proper @dircategory/@direntry.


--- gnustep-base-1.20.0.orig/Documentation/coding-standards.texi
+++ gnustep-base-1.20.0/Documentation/coding-standards.texi
@@ -1,15 +1,12 @@
 \input texinfo  @c -*-texinfo-*-
-@setfilename coding-standards.info
+@setfilename gs-standards.info
 @set DATE 26 Jun 1996
 
-@ifinfo
-@format
-INFO-DIR-SECTION GNUstep
-START-INFO-DIR-ENTRY
-* Coding: (coding-standards).           Coding Standards for GNUstep Libraries
-END-INFO-DIR-ENTRY
-@end format
-@end ifinfo
+@dircategory GNU organization
+@direntry
+* GNUstepStandards: (gs-standards).     Coding Standards for GNUstep
+                                          Libraries.
+@end direntry
 
 @ifinfo
 Copyright @copyright{} 1997-2005 Free Software Foundation
--- gnustep-base-1.20.0.orig/Documentation/manual/manual.texi
+++ gnustep-base-1.20.0/Documentation/manual/manual.texi
@@ -1,18 +1,16 @@
 \input texinfo              @c -*-texinfo-*-
 
 @c %**start of header
-@setfilename manual.info
+@setfilename gs-base.info
 @settitle Objective-C GNUstep Base Programming Manual
 @c %**end of header
 
-@ifinfo
-@format
-INFO-DIR-SECTION GNUstep
-START-INFO-DIR-ENTRY
-* GNUstepBase: (manual).        Objective C Language and GNUstep Base Library
-END-INFO-DIR-ENTRY
-@end format
-@end ifinfo
+@dircategory Software libraries
+@direntry
+* GNUstepBase: (gs-base).       Objective-C Language and GNUstep Base
+                                  Library Programming Manual.
+@end direntry
+
 
 @ifinfo
 Copyright @copyright{} 2001 Free Software Foundation
